async-book/src/execution/executor.md

```diff
@@ -0,0 +1,160 @@
+# Applied: Build an Executor
```

Comment:
Before launching into the example, it'd be good to say a bit more about what an executor *is* and a bit about the properties of the one you're going to build.

Reply:
Done-- let me know what you think